src/community/whats-new/index.md

```diff
@@ -0,0 +1,93 @@
+
+This release introduces new Sass functions to help write `@media` and `@container` queries, mixins and functions whilst still using custom breakpoints or [GOV.UK Frontend’s `$govuk-breakpoints`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-breakpoints) setting.
+
+You can continue using the [`govuk-media-query` mixin](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-media-query) exactly as before - we changed how it works under the hood, but the mixin should behave the same way.
+
+Read the [release notes for v5.13.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.13.0) to see what’s changed.
+
+## September 2025
+
+### We released GOV.UK Frontend 5.12.0
+
+It introduces a new mixin you can use to style the focus state if you’re building your own form input components.
+
+Read the [release notes for v5.12.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.12.0) to see what’s changed.
+
+## August 2025
+
+### We released GOV.UK Frontend 5.11.2
+
+This release includes some JavaScript bug fixes, as well as a fix for thickness of underlines on the Pagination component in Safari.
+
+Read the [release notes for v5.11.2](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.2) to see what changed.
+
+## July 2025
+
+### We released GOV.UK Frontend 5.11.1
+
+It fixes link styles in the Service navigation on inverted backgrounds and makes govuk-shade and govuk-tint more reliable.
+
+Read the [release notes for v5.11.1](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.1) to see what's changed.
+
+## June 2025
+
+### We released GOV.UK Frontend v5.11.0
+
+This release includes improvements to the Service navigation component, making it easier to use on mobile devices and offering an inverse colour option. We also added deprecation warnings for code built with the LibSass library.
+
+Read the [release notes for v5.11.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.0) to see what's changed.
+
+## May 2025
+
+### We released GOV.UK Frontend v4.10.0
+
+This is the first step towards refreshing the GOV.UK brand for users of earlier versions of GOV.UK Frontend.
+
+Read the [release notes for v4.10.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.10.0) to see what's changed.
+
+### We released GOV.UK Frontend v5.10.2
+
+This release is first step towards refreshing the GOV.UK brand.
+
+It includes updates to the:
+
+- [GOV.UK header component](https://design-system.service.gov.uk/components/header/)
+- [GOV.UK footer component](https://design-system.service.gov.uk/components/footer/)
+- [Service navigation component](https://design-system.service.gov.uk/components/service-navigation/)
+- [Cookie banner component](https://design-system.service.gov.uk/components/cookie-banner/)
+
+Read the [release notes for v5.10.2](https://github.com/alphagov/govuk-frontend/releases/tag/v5.10.0) to see what's changed.
```

Comment:
This says 5.10.2 but the link is going to 5.10.0. Should the link text say 5.10.0?